src/components/PageLastUpdated/__tests__/PageLastUpdated.test.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { render } from '@testing-library/react';
+import { PageLastUpdated } from '../index';
+import flatDirectory from '../../../directory/flatDirectory.json';
+
+describe('PageLastUpdated', () => {
+  const directoryData = flatDirectory['/[platform]/how-amplify-works'];
+  const component = <PageLastUpdated directoryData={directoryData} />;
+  it('should render the PageLastUpdated component', async () => {
+    render(component);
+
+    const pageLastUpdatedNode =
+      document.getElementsByClassName('page-last-updated')[0];
+    expect(pageLastUpdatedNode).toBeInTheDocument();
+  });
+
+  it('should show the last updated date', async () => {
+    render(component);
+    const date = new Date(directoryData.lastUpdated).toLocaleDateString(
+      'en-us',
+      {
+        year: 'numeric',
+        month: 'short',
+        day: 'numeric'
+      }
+    );
+
+    const pageLastUpdatedNode =
+      document.getElementsByClassName('page-last-updated')[0];
+    expect(pageLastUpdatedNode.textContent).toBe(`Page updated ${date} `);
```

Comment:
Since you're checking for the existence of the text content, you could just use screen.queryByText here:

```
const lastUpdated = await screen.queryByText(`Page updated ${date} `)
expect(lastUpdated).isNotNull()
```
see: https://stackoverflow.com/questions/76705164/whats-the-difference-between-getbytext-vs-findbytext-vs-querybytext-in-testing